src/python-common/.gitignore

```diff
@@ -1 +1,2 @@
 ceph.egg-info
+/build/
```

Comment:
i have trouble understanding this one. do you create "build" directory here?

Reply:
No, i use our [dev docker system](https://github.com/ricardoasmarques/ceph-dev-docker) and scripts to build Ceph, that's all. After building Ceph the directory looks like:
```
/ceph/src/python-common
├── build
│   ├── bdist.linux-x86_64
│   └── lib
│       └── ceph
│           ├── deployment
│           │   ├── drive_group.py
│           │   ├── __init__.py
│           │   └── ssh_orchestrator.py
│           ├── exceptions.py
│           ├── __init__.py
│           └── tests
│               ├── __init__.py
│               └── test_drive_group.py
├── ceph
│   ├── deployment
│   │   ├── drive_group.py
│   │   ├── __init__.py
│   │   ├── __pycache__
│   │   │   ├── drive_group.cpython-37.pyc
│   │   │   └── __init__.cpython-37.pyc
│   │   └── ssh_orchestrator.py
│   ├── exceptions.py
│   ├── __init__.py
│   ├── __pycache__
│   │   └── __init__.cpython-37.pyc
│   └── tests
│       ├── __init__.py
│       └── test_drive_group.py
├── ceph.egg-info
│   ├── dependency_links.txt
│   ├── not-zip-safe
│   ├── PKG-INFO
│   ├── requires.txt
│   ├── SOURCES.txt
│   └── top_level.txt
├── CMakeLists.txt
├── requirements.txt
├── setup.py
├── tox.ini
└── wheelhouse
    ├── md5
    ├── pip-19.2.1-py2.py3-none-any.whl
    ├── setuptools-41.0.1-py2.py3-none-any.whl
    ├── six-1.12.0-py2.py3-none-any.whl
    └── wheel-0.33.4-py2.py3-none-any.whl

13 directories, 32 files
```

Reply:
@votdev i still don't understand why `build` directory is created in this directory. and i cannot find any clue that your dev docker system and scripts could create it by search "build". see https://github.com/ricardoasmarques/ceph-dev-docker/search?q=build&unscoped_q=build . 

could you please help me understand this?

Reply:
I'm sorry, i can only tell you that the /build directory is there after building Ceph. I don't know how this is created or by whom. I only executed ``setup-ceph.sh`` in my build environment.

Reply:
I have the same problem when building ceph in my docker container, but I'm not sure why this happens :/
I haven't seen that all the time because I copy `/build` from my building container to my development container, but it exists in my build container too.

Reply:
It is created when running `install-deps.sh`.
Maybe its related to openSUSE.

Reply:
The directory is not created anymore with latest master, so i will remove it from .gitignore.